flixel/util/FlxTimer.hx

```diff
@@ -281,12 +309,35 @@ class FlxTimerManager extends FlxBasic
 	public function completeAll():Void
```

Comment:
It's more complex because docs about these methods says different things:
- in flxtween it says that onComplete callback is called only once
- in flxtimer - onComplete callback will be called `loopsLeft` times.

With your change there is should be `onLoopFinished()` call, because timer's `update()` method had been changed and doesn't contain this logic anymore